packages/types/src/index.ts

```diff
@@ -226,6 +241,7 @@ export interface ExternalSelect extends Action {
 
 export interface MultiExternalSelect extends Action {
   type: 'multi_external_select';
+  // event to return option data. so de-facto this is necessary
```

Comment:
I'm not certain I understand what this comment means! Is this a note to self?

Reply:
LOL it was a much longer comment that I had before that I then decided to remove _which I clearly didn't do a great job of_ 😆